### PR TITLE
bnsd: add a no-op migration for local-iov-devnet network

### LINKED ISSUE #1085: Create no-op migration for local-iov-devnet
In order to build a test for https://github.com/iov-one/iov-core/issues/1341 we need a migration that is successfully executed on chain ID `local-iov-devnet`. This can be a migration that does nothing.

Required signer is "cond:gov/rule/0000000000000002" which we can call `devnetRule2`. This has nothing to do with "cond:gov/rule/0000000000000002" from mainnet because it is a different genesis file.

### PR DESCRIPTION
**Pull request by husio**
resolve #1085

!nochangelog

### CODE REVIEW

File: cmd/bnsd/x/account/init.go [hunk 1]
```diff
@@ -57,6 +63,7 @@ func (*Initializer) FromGenesis(opts weave.Options, params weave.GenesisParams,
 		account := Account{
 			Metadata: &weave.Metadata{Schema: 1},
 			Domain:   d.Domain,
+			Owner:    d.Admin,
```
Comment (webmaster128) [inline cmd/bnsd/x/account/init.go]: Huh? Why is this unrelated change in here?

Comment (husio) [reply cmd/bnsd/x/account/init.go]: Correct.
I have noticed this issue while testing manually. It's not related to no-op migration.

Comment (webmaster128) [reply cmd/bnsd/x/account/init.go]: okay, thanks. This looks like a bugfix of the `account` module that deserves a CHANGELOG entry

Comment (husio) [reply cmd/bnsd/x/account/init.go]: This change does not belong here if backend was a team and not one person. Because I am alone right now, I allow myself for a big of controlled chaos :smiling_imp: 

I do not create changelog entries for all changes, only for those that are subjectively considered important. Do you think this change deserves a changelog entry?

Comment (webmaster128) [reply cmd/bnsd/x/account/init.go]: > Because I am alone right now, I allow myself for a big of controlled chaos 😈

Yeah, hard to get used to that but fine for me.

> are subjectively considered important. Do you think this change deserves a changelog entry?

The diff looks like creating a chain with `account` config in genesis is broken. This is very important for local development chains as soon as we start using the new functionality. A client dev needs to know: starting a new chain with bnsd 0.24.x and accounts config is broken, i.e. 0.25.0 must be used instead.
